src/guide/extras/reactivity-in-depth.md

```diff
@@ -8,7 +8,7 @@ import SpreadSheet from './demos/SpreadSheet.vue'
 
 # Reactivity in Depth
 
-One of Vue’s most distinct features is the unobtrusive reactivity system. Component state are reactive JavaScript objects. When you modify them, the view updates. It makes state management simple and intuitive, but it’s also important to understand how it works to avoid some common gotchas. In this section, we are going to dig into some of the lower-level details of Vue’s reactivity system.
+One of Vue’s most distinct features is the unobtrusive reactivity system. Component state is reactive JavaScript objects. When you modify them, the view updates. It makes state management simple and intuitive, but it’s also important to understand how it works to avoid some common gotchas. In this section, we are going to dig into some of the lower-level details of Vue’s reactivity system.
```

Comment:
Hey -- thanks for including me! I'm honored. Got you on "composed". I think "consists of" works just fine. 

I do think you want "distinctive" rather than "distinct" because you're highlighting a defining product feature, not simply saying it's an easily identifiable part.

Reply:
Hi. I've just updated my earlier suggestion to include the word *distinctive*. If you're happy with it could you click the button underneath the diff to commit it to your branch? (I'm assuming you'll see a button, GitHub permissions may decide otherwise). Then we can get this merged into `main`. Thanks.